src/components/search/ClimbSearchByName.tsx

```diff
@@ -3,47 +3,57 @@ import { useRouter } from 'next/router'
 import { typesenseSearch } from '../../js/typesense/TypesenseClient'
 import { Autocomplete } from './Autocomplete'
 import { SearchByNameTemplate } from './ResultTemplates'
+import { debounced } from '../../js/utils'
 
 export const ClimbSearchByName = ({ isMobile = true, placeholder = 'Try \'Levitation 29\', \'technical crimpy\', or \'Lynn Hill\'' }: {isMobile?: boolean, placeholder?: string}): JSX.Element => {
   const router = useRouter()
+
   useEffect(() => {
```

Comment:
this useEffect is causing the autocomplete to fail in some situations. Example: click on "Climb Search", enter value, get result, change tab to "Places to Climb", change tab back to "Climb Search", autocomplete will not produce results.